pkg/k8s/kubeconfig/const.go

```diff
@@ -0,0 +1,6 @@
+package kubeconfig
+
+const (
+	// OktetoUseStaticKubetokenEnvVar is used to opt in to use static kubetoken
```

Comment:
this const needs to be public in order to use it in e2e test so moving it here because import `context` pkg can have a bigger impact. This way is more isolated

Reply:
I don't know if we should create a kubeconfig package just to have a variable inside. We are using it just in the context package. Should it live there and import it in the tests?

Reply:
it only creates a new `const.go` file but yeah for now I think it is more reasonable your point

Reply:
done 43f5e125b93ca0e03eb19bb9659691f6f44e3110